src/laser_geometry.cpp

```diff
@@ -302,6 +302,8 @@ const boost::numeric::ublas::matrix<double>& LaserProjection::getUnitVectors_(do
       {
         ranges(i,0) = (double) (scan_in.ranges[i] > range_cutoff
                           ? range_cutoff : scan_in.ranges[i]);
+        ranges(i,0) = (double) (scan_in.ranges[i] < scan_in.range_min
```

Comment:
Even better, use `boost::clamp`: http://www.boost.org/doc/libs/1_53_0/boost/algorithm/clamp.hpp (`std::clamp` in C++17: http://en.cppreference.com/w/cpp/algorithm/clamp)

``` cpp
ranges(i,0) = boost::clamp(scan_in.ranges[i], scan_in.range_min, range_cutoff);
```